README.md

```diff
@@ -1,24 +1,47 @@
-# README
+# Blog app
+This is an app where the user can create a post and on it, exist the posibility to do comments about it.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+## Built With:
 
-* Ruby version
+- Ruby on Rails
+- SQL
 
-* System dependencies
+## Getting Started
+- To get a local copy of this project clone this repository or download the Zip folder:
+git clone https://github.com/BregornOriginal/blog-app.git
 
-* Configuration
+- To get started, In the root directory run:
+gem install bundler
 
-* Database creation
+- Then run:
+bundle install
 
-* Database initialization
+- Run ruby app in the Command Line:
+ruby main.rb
```

Comment:
- [ ] Kindly change your command for running the rails app, its incorrect, we use rails s/ server to run rails app.

Reply:
Thanks!